pkgs/by-name/ha/harlequin/package.nix

```diff
@@ -79,8 +79,8 @@ python3Packages.buildPythonApplication rec {
       "test_connect_extensions"
       "test_connect_prql"
     ]
-    ++ lib.optionals (stdenv.hostPlatform.system == "aarch64-darwin") [
-      # Test incorrectly tries to load a dylib compiled for x86_64
+    ++ lib.optionals stdenv.hostPlatform.isAarch64 [
```

Comment:
```suggestion
    ++ lib.optionals (!stdenv.hostPlatform.isx86_64) [
```

Reply:
The change isn't applied?

Reply:
Oops, I reverted it with a forced pushed from an outdated branch because I wanted to restart the failing GHA check.